gulpfile.js

```diff
@@ -63,7 +62,7 @@ gulp.task('build:bundle', [
 });
 
 // Build app
-gulp.task('build:app', [], function()
+gulp.task('build:app', ['build:lint'], function()
```

Comment:
We removed it to get a better performance for the watcher. I will add another task to fix this problem. For now we can merge it to get the fix.